src/account.ts

```diff
@@ -66,6 +66,21 @@ export abstract class Account {
         calldata?: StringMap,
         options?: InvokeOptions
     ): Promise<InvokeResponse> {
+        let maxFee: starknet.FeeEstimation;
+        if (options?.maxFee && options?.overhead) {
+            const msg = "Both maxFee and overhead cannot be specified";
+            throw new StarknetPluginError(msg);
+        }
+
+        if (!options?.maxFee) {
```

Comment:
What if maxFee is 0? We should use it in that case and only estimate if it's null or undefined

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/206/commits/253aeb445b361f5e6f81c600166c7f916ef2a545